MahApps.Metro/Properties/AssemblyInfo.cs

```diff
@@ -18,5 +18,5 @@
 [assembly: AssemblyTitleAttribute("MahApps.Metro")]
 [assembly: AssemblyDescriptionAttribute("Toolkit for creating Metro styled WPF apps")]
 [assembly: AssemblyProductAttribute("MahApps.Metro 1.1.3-ALPHA")]
-
+[assembly: AssemblyCompany("MahApps Metro")]
```

Comment:
@punker76 I think this could be simplified to `MahApps` or even `MahApps Team` - let me know what your thoughts are on this.